org.eclipse.gef.dot.tests/src/org/eclipse/gef/dot/tests/test.properties

```diff
@@ -0,0 +1 @@
+dotpath=/usr/local/bin/dot
```

Comment:
Is this meant to be hardcoded?

Reply:
We have already removed the `test.properties` file after the merge. That had been checked in by accident.